Week-1/html/exercise_1.2/music_player/css/currentsong.css

```diff
@@ -0,0 +1,37 @@
+.album-wrapper{
+  display:flex;
```

Comment:
display: flex associated with all .album elements, can we make it common ?

Reply:
created common class for display flex